FortnoxAPILibrary/UrlRequestBase.cs

```diff
@@ -499,9 +501,15 @@ internal FortnoxError.ErrorInformation HandleException(WebException we)
             {
                 this.httpStatusCode = response.StatusCode;
 
-                if (this.httpStatusCode == HttpStatusCode.InternalServerError)
+                switch (httpStatusCode)
                 {
-                    throw we;
+                    case HttpStatusCode.InternalServerError:
+                        throw we;
+                    case HttpStatusCode.TooManyRequests:
+                        throw new FortnoxTooManyRequestsException(we.Message, we)
+                        {
+                            Code = httpStatusCode
```

Comment:
saving code is extra here, because we now that FortnoxTooManyRequestsException == 429

Reply:
Resolved